Floofbot/Modules/Utilities.cs

```diff
@@ -99,5 +102,44 @@ public async Task RepeatMessage([Remainder] string message =null)
                 await Context.Channel.SendMessageAsync("Usage: `.say [message]`");
             }
         }
+
+        [Command("serverinfo")]
+        [Summary("Returns information about the current server")]
+        public async Task ServerInfo()
+        {
+            SocketGuild guild = Context.Guild;
+            int numberTextChannels = guild.TextChannels.Count;
+            int numberVoiceChannels = guild.VoiceChannels.Count;
+            int daysOld = Context.Message.CreatedAt.Subtract(guild.CreatedAt).Days;
+            string createdAt = $"Created {guild.CreatedAt.DateTime.ToShortDateString()}. " +
+                               $"That's " +
+                               ((daysOld > 1) ? $"over {daysOld} days" : $"{daysOld} day") +
```

Comment:
What about 0 _days_ now? :^) 
You might want to create a new string variable separately first to avoid createdAt being a long string full of ternaries.